knowledge_base/serverless_job/README.md

```diff
@@ -0,0 +1,38 @@
+# Serverless job
+
+This example demonstrates how to define and use a serverless job in a Databricks Asset Bundle.
```

Comment:
Nice!

Maybe this could link to https://docs.databricks.com/workflows/jobs/how-to/use-bundles-with-jobs.html#configure-a-job-that-uses-serverless-compute for further reading?

Reply:
Added.